docpages/building/linux.md

```diff
@@ -36,9 +37,24 @@ Once installed to the `/usr/local` directory, you can make use of the library in
 g++ -std=c++17 mydppbot.cpp -o dppbot -ldpp
 ```
 
+If you are on **Arch Linux**, you will need to add `/usr/local` to your environment variables:
+
+```bash
+export LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
+export CPLUS_INCLUDE_PATH=/usr/local/include:$CPLUS_INCLUDE_PATH
+```
+
+However, if you don't want to change your environment settings, you can use these flags to compile:
+
+```bash
+g++ -std=c++17 -L/usr/local/lib -I/usr/local/include -ldpp mydppbot.cpp -o dppbot
```

Comment:
Sorry, change this to `g++ -std=c++17 -L/usr/local/lib -I/usr/local/include mydppbot.cpp -o dppbot  -ldpp`, forgot linux demands `-ldpp` at the end.